aten/src/ATen/native/native_functions.yaml

```diff
@@ -487,6 +487,10 @@
 - func: all.out(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
   structured: True
+  precomputed:
+    elements: int64_t dim_
+    replace:
+      - dim -> dim_
```

Comment:
I guess it's because this would look weird?

Reply:
Yeah, that's why I included the underscore.